data/campaigns/Under_the_Burning_Suns/scenarios/02_Across_the_Harsh_Sands.cfg

```diff
@@ -1430,17 +1432,42 @@
             y=13-19
         [/filter]
 
+#ifdef EASY
+        [scroll_to]
+            x,y=10,14
+        [/scroll_to]
+#else
         [terrain]
             terrain=Dd
             x=10,10,11,11,11,12,12
             y=14,15,15,16,17,15,16
         [/terrain]
-
+#endif
         {CHECK_EXPLORER}
+#ifdef EASY
+        [message]
+            speaker=$explorer.id
+            message= _ "Phew, thank goodness that this oasis is here! After hearing about the bandits in this area, I was beginning to worry that they had created a mirage to trick thirsty travelers such as ourselves!"
+        [/message]
+        [message]
+            speaker=Nym
+            message= _ "I know these bandits are evil, but do you really think they'd stoop so low as to get their victims' hopes up before crushing them?"
+            image=portraits/nym_moody.png
+        [/message]
+        [message]
+            speaker=Zhul
+            message= _ "It has been known to happen before."
+        [/message]
+        [message]
+            speaker=Kaleh
+            message= _ "Truly, there are some depraved and cruel individuals in this world."
+        [/message]
+#else
```

Comment:
I think this kind of dialog makes more sense when we actually find it's a mirage, instead of when we don't have one.

Reply:
also, it's important to put *something* there to explain why the mirage thing isn't happening, when players might expect it to be

Reply:
Okay, that explains why it's there fore EASY, but it might make sense to have a similar dialogue (a bit less hypothetical) also for the other difficulties – i.e. when you find it's a mirage, do some speculation whether the bandits have set this up intentionally.

Reply:
Added some additional dialogue in [66a70fa](https://github.com/wesnoth/wesnoth/pull/8421/commits/66a70fad038af1e557e864037c8ca6cd48a7c7e6).